manifest.toml

```diff
@@ -57,7 +57,6 @@ ram.runtime = "50M"
     [resources.install_dir]
 
     [resources.data_dir]
-    dir = "/var/lib/__APP__"
```

Comment:
Why remove this line ? Is it just because it explicits the default value ?

Reply:
This introduced the bug described in the header of this PR ;)